README.md

```diff
@@ -333,6 +333,9 @@ sock.ev.on('messages.upsert', ({ messages }) => {
 > [!NOTE]
 > This example includes basic auth storage too
 
+> [!NOTE]
+> For reliable serialization of the authentication state, especially when storing as JSON, always use the BufferJSON utility.
+
```

Comment:
This needs to be added to the guide too

Reply:
encourage the use of decodeAndHydrate in the guide too with the breaking changes for the proto